engine/src/main/java/org/terasology/rendering/dag/nodes/LateBlurNode.java

```diff
@@ -67,11 +69,14 @@ protected void setupConditions(Context context) {
         renderingConfig = context.get(Config.class).getRendering();
         renderingConfig.subscribe(RenderingConfig.BLUR_INTENSITY, this);
         requiresCondition(() -> renderingConfig.getBlurIntensity() != 0);
+        worldRenderer = context.get(WorldRenderer.class);
     }
 
     @Override
     public void propertyChange(PropertyChangeEvent event) {
         updateBlurRadius();
+        // Changing the blurRadius can potentially enable/disable the Node, meaning we have to refresh the taskList.
```

Comment:
This is not obvious from the code in this method and in updateBlurRadius(). Can you be more explicit on why this is possible? Where exactly is the node disabled?

Reply:
Based on the explanation I gave below, I only update the value of blur here. When refreshing task list, the WR will check if this Node is enabled(), which will in turn call checkConditions to ensure all conditions are met, which is where the magic happens.